src/components/userGroups/Group.tsx

```diff
@@ -14,13 +14,18 @@ import './UserGroup.scss'
 import { toast } from 'react-toastify'
 import AppPermissions from './AppPermissions'
 import { ACCESS_TYPE_MAP, SERVER_MODE } from '../../config'
+import { PermissionType } from '../apiTokens/authorization.utils'
 import { mainContext } from '../common/navigation/NavigationRoutes'
 import { ReactComponent as Warning } from '../../assets/icons/ic-warning.svg'
 import { excludeKeyAndClusterValue } from './K8sObjectPermissions/K8sPermissions.utils'
 
+enum PermissionTypeEnum {
```

Comment:
Please move this to type file

Reply:
will handle in the active/inactive story